pkg/asset/manifests/content/bootkube/cvo-overrides.go

```diff
@@ -0,0 +1,59 @@
+package bootkube
+
+import (
+	"text/template"
+)
+
+var (
+	// CVOOverrides is the constant to represent contents of cvo-override.yaml file
+	// This is a gate to prevent CVO from installing these operators which is conflicting
+	// with already owned resources by tectonic-operators.
+	// This files can be dropped when the overrides list becomes empty.
+	CVOOverrides = template.Must(template.New("cvo-override.yaml").Parse(`
+apiVersion: clusterversion.openshift.io/v1
+kind: CVOConfig
+metadata:
+  namespace: openshift-cluster-version
+  name: cluster-version-operator
+upstream: http://localhost:8080/graph
+channel: fast
+clusterID: {{.CVOClusterID}}
+overrides:
+- kind: Deployment                    # this conflicts with kube-core-operator
+  namespace: openshift-core-operators
+  name: openshift-cluster-kube-apiserver-operator
+  unmanaged: true
+- kind: Deployment                    # this conflicts with kube-core-operator
+  namespace: openshift-core-operators
+  name: openshift-cluster-kube-scheduler-operator
+  unmanaged: true
+- kind: Deployment                    # this conflicts with kube-core-operator
+  namespace: openshift-core-operators
+  name: openshift-cluster-kube-controller-manager-operator
+  unmanaged: true
+- kind: Deployment                    # this conflicts with kube-core-operator
+  namespace: openshift-core-operators
+  name: openshift-cluster-openshift-apiserver-operator
+  unmanaged: true
+- kind: Deployment                    # this conflicts with kube-core-operator
+  namespace: openshift-core-operators
+  name: openshift-cluster-openshift-controller-manager-operator
+  unmanaged: true
+- kind: Deployment                    # this conflicts with kube-core-operator
+  namespace: openshift-cluster-network-operator
+  name: cluster-network-operator
+  unmanaged: true
+- kind: Deployment                    # this conflicts with tectonic-ingress-controller-operator
```

Comment:
Not that it matters now, but this doesn't actually conflict

Reply:
@ironcladlou if that'd the case feel free to open a PR to drop this override.